python/test/t_KDTree_std.py

```diff
@@ -2,16 +2,40 @@
 
 from __future__ import print_function
 import openturns as ot
+import numpy as np
+import os
 
 ot.TESTPREAMBLE()
 
 
-sample = ot.Normal(3).getSample(10)
+sample = ot.Normal(3).getSample(1000)
 tree = ot.KDTree(sample)
 print('tree=', tree)
 
-test = ot.Normal(3).getSample(20)
-for i in range(len(test)):
-    index = tree.getNearestNeighbourIndex(test[i])
-    neighbour = tree.getNearestNeighbour(test[i])
-    print('Nearest neighbour of', test[i], '=', neighbour, 'index=', index)
+test = ot.Normal(3).getSample(100)
+
+sample_np = np.array(sample)
+test_np = np.array(test)
+
+def nearest_debug(x):
+    global sample_np
+    a = np.sum(np.square(sample_np - x), axis=1)
+    return np.argmin(a)
+
+def nearest_debug_indices(x):
+    global sample_np
+    a = np.sum(np.square(sample_np - x), axis=1)
+    return a.argsort()
+
+neighbourIndices = tree.getNearestNeighbourIndex(test)
+
+neighbourIndices_np = map(nearest_debug, test)
+
+if neighbourIndices != neighbourIndices_np:
```

Comment:
On the autobuilder:
TypeError: Object passed as argument is not convertible to an Indices

This must be because map does not return a list in python3
I suggest `neighbourIndices_np = [nearest_debug_indices(x) for x in test]` which is also more readable than map

Reply:
Indeed, I always forgot about that :-/ Will fix it.